test/unit/api/emit.js

```diff
@@ -1,5 +1,4 @@
 import emit from '../../../src/api/emit';
```

Comment:
This file was refactored and tests reordered to test simpler and dependent functionality first before it's used later in the tests.